### DISCUSSION TITLE
font size

### DISCUSSION THREAD

**Discussion by kalle07** [h2oai/h2o-llmstudio#569] [Q&A]
any idea where i can change ???

**Comment by maxjeblick** [comment]
You could zoom in/zoom out within the browser window, font and layout will adjust automatically.

**Comment by kalle07** [comment]
oh sry i use the stand alone version ...  so there i mouse weel not work and i see no zoom button ...

**Reply by maxjeblick** [reply]
What I meant is to change the page size within the browser using Ctrl and + and Ctrl and -, respectively.

**Comment by kalle07** [comment]
yes that what i mean too .. in the stand alone version LM Studio.exe , it doesnt work !  ;)

**Reply by maxjeblick** [reply]
What do you mean exactly with standalone exe version? H2O LLM Studio does not provide an .exe file.  
The app will run in the respective browser where you open it. If you cannot zoom in/zoom out in your browser, it looks like a browser issue.

**Comment by kalle07** [comment]
oh sry it give LM and LLM  ;)
you can forget all . . .
